### PR TITLE
commerce frontend js cleaned up

### PR DESCRIPTION
**Pull request by fabio-d-m**
Also created a dumb boilerplate component

### CODE REVIEW

**Comment by marco-leo** [comment]
ci:test

**Comment by marco-leo** [comment]
ci:test

**Comment by ethanbustad** [comment]
@FabioDiegoMastrorilli running `gw formatSource` should auto-remove the file that the validation task is failing on.

File: commerce-frontend-js/dev/public/example.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8" />
+
+	<meta
+		content="width=device-width, initial-scale=1, shrink-to-fit=no"
+		name="viewport"
+	/>
+	<!-- <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/clay/lib/css/atlas.css" />
```
Comment (ethanbustad) [inline commerce-frontend-js/dev/public/example.html:10]: This comment seems unnecessary :)

File: commerce-frontend-js/dev/public/index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8" />
+
+	<meta
+		content="width=device-width, initial-scale=1, shrink-to-fit=no"
+		name="viewport"
+	/>
+	<!-- <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/clay/lib/css/atlas.css" />
```
Comment (ethanbustad) [inline commerce-frontend-js/dev/public/index.html:10]: also unnecessary

File: commerce-frontend-js/src/main/java/com/liferay/commerce/frontend/js/servlet/taglib/CommerceFrontendJsDynamicInclude.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	@Override
+	public void include(
+			HttpServletRequest httpServletRequest,
+			HttpServletResponse httpServletResponse, String key)
+		throws IOException {
+
+		PrintWriter printWriter = httpServletResponse.getWriter();
+
+		printWriter.print(
+			"<link rel=\"stylesheet\" type=\"text/css\" href=\"/o" +
```
Comment (ethanbustad) [inline commerce-frontend-js/src/main/java/com/liferay/commerce/frontend/js/servlet/taglib/CommerceFrontendJsDynamicInclude.java:46]: _Ideally_, the attributes in this `<link>` tag would be sorted alphabetically. Not the hugest deal embedded in java code, but would be nice.

**Review by ethanbustad — COMMENTED** [review]
I don't think I understand all the context here.... I made a couple comments that would be "nice-to-have"s, but if they don't make sense please let me know!

File: commerce-frontend-js/src/main/java/com/liferay/commerce/frontend/js/servlet/taglib/CommerceFrontendJsDynamicInclude.java (hunk 3 above)
Comment (alecsloan) [reply commerce-frontend-js/src/main/java/com/liferay/commerce/frontend/js/servlet/taglib/CommerceFrontendJsDynamicInclude.java:46]: @FabioDiegoMastrorilli I fixed this one but didn't touch the above comments

**Comment by ethanbustad** [comment]
Alec made these fixes and sent to https://github.com/liferay/com-liferay-commerce/pull/1507.

**Pull request closed without merging**